annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/graph/ConflictGraph.java

```diff
@@ -74,7 +74,7 @@ public Node addNodeToVertices(Fix fix) {
    * in the same group. A greedy algorithm is used to find the solution.
    */
   // TODO: Remove SuppressWarnings below later.
-  @SuppressWarnings("JdkObsolete")
+  @SuppressWarnings({"JdkObsolete", "unchecked"})
```

Comment:
Why not just fix the error instead of suppressing?

Reply:
I didn't know how I can resolve the warning for creating array of `LinkedLists`. Since I was confident that there will be no problem with the code I just silenced it. 

However, I just changed it to use List of Lists 6517fba69335d5a5e7038d61722e0065a5eec73a, I don't think this will impact the performance. But if you think this will actually impact the performance, please let me know and I will undo this. Thank you.